### PR TITLE
Version history

### PR DESCRIPTION
**Pull request by thescubageek**
Adds a route (etc) for showing the version history log. You can see a list of the version changes by clicking on the version number in the breadcrumb nav.

### CODE REVIEW

File: app/controllers/versions.coffee [hunk 1]
```diff
@@ -0,0 +1,8 @@
+`import Ember from 'ember'`
+
+VersionsController = Ember.Controller.extend
+  versionHtml: ( ->
```
Comment (thescubageek) [inline app/controllers/versions.coffee]: This isn't used, probably can be removed

File: app/routes/versions.coffee [hunk 2]
```diff
@@ -0,0 +1,11 @@
+`import Ember from 'ember'`
+`import ResetScrollMixin from '../mixins/reset-scroll'`
+
+inflector = Ember.Inflector.inflector
```
Comment (thescubageek) [inline app/routes/versions.coffee]: This is probably unnecessary

File: config/environment.js [hunk 3]
```diff
@@ -25,7 +25,7 @@ module.exports = function(environment) {
     ENV.APP.LOG_TRANSITIONS = true;
     // ENV.APP.LOG_TRANSITIONS_INTERNAL = true;
     // ENV.APP.LOG_VIEW_LOOKUPS = true;
-    ENV.APP.host = 'http://localhost:3000';
+    ENV.APP.host = 'http://localhost:5050';
```
Comment (thescubageek) [inline config/environment.js]: Need to roll this back to 3000

**Comment by thescubageek** [comment]
@levibrown Got it all working with the markdown handlebars etc, mind taking a look?

**Comment by levibrown** [comment]
Looks good. I would like to see files that we touch converted to standard js/ es6. would you mind converting those?  would definitely help out with this: https://github.com/G5/orion-cms/pull/20 .

Also, you should add a unit test for the helper you created. should be really easy.

**Comment by thescubageek** [comment]
@levibrown New files have been decaffinated.... working on the helper spec

**Comment by thescubageek** [comment]
Added the helper spec....

**Pull request merged**